src/cmd/manage_nsfs.js

```diff
@@ -83,7 +83,10 @@ async function main(argv = minimist(process.argv.slice(2))) {
 }
 
 async function bucket_management(action, user_input) {
-    const data = await fetch_bucket_data(action, user_input);
+    let data;
+    if (action !== ACTIONS.LIST && action !== ACTIONS.STATUS) {
```

Comment:
for list it's great, for status I would go with the general flow

Reply:
Changed, thanks.